crates/runtime/src/instance/allocator/pooling.rs

```diff
@@ -1341,7 +1341,9 @@ mod test {
         assert_eq!(instances.offsets.num_defined_tables, 1);
         assert_eq!(instances.offsets.num_defined_memories, 1);
         assert_eq!(instances.offsets.num_defined_globals, 0);
-        assert_eq!(instances.instance_size, 4096);
+        // As of April 2021, the instance's size is largely below the size of a single page, so
```

Comment:
s/instance/instance struct/? (I read it the other way and was puzzled briefly)

Reply:
done!